### PR TITLE
[GPU] Fix coverity errors

### PR DESCRIPTION
**Pull request by yeonbok**
### Details:
 - Fix coverity issues

### Tickets:
 - 79245
 - 79241
 - 79259

### CODE REVIEW

File: src/plugins/intel_gpu/src/graph/program_node.cpp [hunk 1]
```diff
@@ -25,7 +25,7 @@
 using namespace cldnn;
 
 program_node::program_node(std::shared_ptr<primitive> prim, program& prog)
-    : desc(prim), myprog(prog), org_id(prim->id) {
+    : desc(prim? prim : nullptr), myprog(prog), org_id(prim ? (prim->id) : 0) {
```
Comment (lznamens) [inline src/plugins/intel_gpu/src/graph/program_node.cpp:28]:
```suggestion
    : desc(prim ? prim : nullptr), myprog(prog), org_id(prim ? (prim->id) : 0) {
```

**Review by lznamens — APPROVED** [review]
LGTM.

File: src/plugins/intel_gpu/src/graph/program_node.cpp (hunk 1 above)
Comment (yeonbok) [reply src/plugins/intel_gpu/src/graph/program_node.cpp]: Applied , thanks!

File: src/plugins/intel_gpu/src/graph/include/program_node.h [hunk 2]
```diff
@@ -68,7 +68,7 @@ struct fused_primitive_desc {
     std::map<primitive_id, size_t> fused_deps;
     size_t total_num_deps = 0;
     activation_func activation;
-    activation_additional_params activation_params;
+    activation_additional_params activation_params = {};
```
Comment (vladimir-paramuzov) [inline src/plugins/intel_gpu/src/graph/include/program_node.h:71]: Shouldn't it be = {0.f, 0.f} ?

File: src/plugins/intel_gpu/src/graph/program_node.cpp [hunk 3]
```diff
@@ -25,7 +25,7 @@
 using namespace cldnn;
 
 program_node::program_node(std::shared_ptr<primitive> prim, program& prog)
-    : desc(prim), myprog(prog), org_id(prim->id) {
+    : desc(prim ? prim : nullptr), myprog(prog), org_id(prim ? (prim->id) : 0) {
```
Comment (vladimir-paramuzov) [inline src/plugins/intel_gpu/src/graph/program_node.cpp:28]: desc(prim)

Comment (yeonbok) [reply src/plugins/intel_gpu/src/graph/program_node.cpp]: I made too much fixed. Updated, thanks!

File: src/plugins/intel_gpu/src/graph/include/program_node.h (hunk 2 above)
Comment (yeonbok) [reply src/plugins/intel_gpu/src/graph/include/program_node.h]: Updated, thanks.

**Pull request merged**